CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [#247](https://github.com/nf-core/mag/pull/247) - Add ancient DNA subworkflow
 - [#263](https://github.com/nf-core/mag/pull/263) - Add MaxBin2 as second contig binning tool
 - [#284](https://github.com/nf-core/mag/pull/285) - Add AdapterRemoval2 as an alternative read trimmer
+- [#284](https://github.com/nf-core/mag/pull/291) - Add DAS_Tool for bin refinement
```

Comment:
```suggestion
- [#284](https://github.com/nf-core/mag/pull/291) - Add DAS Tool for bin refinement
```

Reply:
should we stick to this for consistency? would also make searching easier

Reply:
i have no strong opinion, I can replce it everwhree in the docs in case you've not picked it up everwhere.